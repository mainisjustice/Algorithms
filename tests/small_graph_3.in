6
0 3 5 0 0 4
3 0 5 3 4 7
5 5 0 4 0 7
0 3 4 0 1 2
0 4 0 1 0 9
4 7 7 2 9 0