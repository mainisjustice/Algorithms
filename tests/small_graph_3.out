0 3 5 6 7 4 
3 0 5 3 4 5 
5 5 0 4 5 6 
6 3 4 0 1 2 
7 4 5 1 0 3 
4 5 6 2 3 0 